backend/scripts/server

```diff
@@ -15,7 +15,7 @@ FLASK_ENV=development
 # Start Worker and Server. Whatchdog detects any change to the .py files and restarts celery
 poetry run watchmedo auto-restart -d . -p '*.py' -- celery worker -E -B -A inspirehep.celery -l INFO & pid_celery=$!
 
-poetry run gunicorn --reload -b 0.0.0.0:5000  --access-logfile "-" --error-logfile - inspirehep.wsgi:application & pid_server=$!
+poetry run gunicorn --reload -b 0.0.0.0:8000  --access-logfile "-" --error-logfile - inspirehep.wsgi:application & pid_server=$!
```

Comment:
Why is this needed?

Reply:
to match the ``SERVER_NAME`` otherwise the requests are not working